vmdb/db/migrate/20150630025128_create_tenant.rb

```diff
@@ -0,0 +1,17 @@
+class CreateTenant < ActiveRecord::Migration
+  def change
+    create_table :tenants do |t|
+      t.string :domain
+      t.string :subdomain
+      t.string :customer_name
+      t.string :vmdb_name
```

Comment:
@dclarizio the ui uses `session[:vmdb_name]` and `session[:customer_name]` (taken from `vmdb.yml/application/vmdb_name` and `vmdb.yml/application/company`) could you explain the difference to us?